src/profile.ts

```diff
@@ -145,13 +141,13 @@ export class Profile {
         const idx = await resolveIdxProfile(
           formatCAIP10Address(address, "eip155", config.network.chainId), config
         );
-        return { address, ens: null, idx };
+        return { address, idx: idx || undefined };
```

Comment:
I think this is a good place to use the nullish coalescing operator, since we are checking for nullish values.

```suggestion
        return { address, idx: idx ?? undefined };
```